src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -1503,35 +1481,23 @@ private bool CanExecuteActiveCode()
             return _evaluator.CanExecuteCode(input);
         }
 
-        #endregion
+#endregion
 
-        #region Buffers, Spans and Prompts
-        private ITrackingSpan CreateStandardInputPrompt()
+#region Buffers, Spans and Prompts
```

Comment:
I'll leave the methods as is for now so it's clear which prompt we're expecting in each case. Logged #4779 to add distinct methods to IInteractiveEvaluator.